libs/dh/admin/feature-invite-user-modal/src/lib/dh-invite-user-modal.component.html

```diff
@@ -77,7 +77,7 @@
             type="email"
             name="userRole"
           />
-          <span wattSuffix>@ok.dk</span>
+          <span wattSuffix>@{{ organizationDomain$ | push }}</span>
```

Comment:
question: in case `organizationDomain$` is `null/undefined`, the input field will look `<empty>@<empty>`. Should we instead show an em-dash if `organizationDomain$` is `null/undefined`, e.g `<empty>@—` ?